lib/Http/RequestHelper.php

```diff
@@ -187,4 +195,22 @@ public function anonymizeIp(string $ip)
 
         return $aip;
     }
+
+    /**
+     * @param string $uri
+     * @return Request
+     */
+    public function createRequestWithContext($uri = '/') {
+        $request =  Request::create(
+            $this->requestContext->getScheme().'://'.$this->requestContext->getHost().$this->requestContext->getBaseUrl().$uri,
+            $this->requestContext->getMethod(),
+            $this->requestContext->getParameters(), [], [],
+            [
+            'SCRIPT_FILENAME' => $this->requestContext->getBaseUrl(),
```

Comment:
Is it really necessary to set `SCRIPT_FILENAME` and `SCRIPT_NAME` ? 
If so, we should probably set them to the right values, right?

Reply:
you are right, it is not required. so removed it.